src/styles/blocks/card.scss

```
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: $color-white;
+  border: 1px solid #f3f3f3;
```

Create a variable for all colors
```suggestion
  border: 1px solid $c-color;
```